akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala

```diff
@@ -32,7 +32,8 @@ object TwitterStreamQuickstartDocSpec {
   //#model
 
   //#tweet-source
-  val tweets: Source[Tweet, NotUsed] //#tweet-source
+  val tweets: Source[Tweet, NotUsed]
+  //#tweet-source
```

Comment:
This has been carefully restored, but scalariform will break it with any chance it gets (i.e. during every compilation). We should definitely fix the code formatting automatism …

(in the reformatted form the `//#tweet-source` snippet is not found by sphinx)